src/imp/openssl.rs

```diff
@@ -115,6 +119,7 @@ pub enum Error {
     Ssl(ssl::Error, X509VerifyResult),
     EmptyChain,
     NotPkcs8,
+    UnsupportedTls13,
```

Comment:
This would be a breaking change, but I don't see a good compatible way to define this error with existing variants.